suite-native/module-send-receive/src/components/ReceiveTextHint.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { Button, VStack, Text } from '@suite-native/atoms';
+
+type ReceiveTextHintProps = {
+    onShowAddress(): void;
+};
+
+const hintWrapperStyle = prepareNativeStyle(utils => ({
+    marginTop: utils.spacings.medium,
+    alignItems: 'center',
+    textAlign: 'center',
+}));
+
+const hintTextStyle = prepareNativeStyle(_ => ({
+    textAlign: 'center',
+}));
+
+export const ReceiveTextHint = ({ onShowAddress }: ReceiveTextHintProps) => {
+    const { applyStyle } = useNativeStyles();
+    return (
+        <VStack spacing="large">
+            <VStack spacing="medium" style={applyStyle(hintWrapperStyle)}>
+                {/* TODO: change the texts when the final design is ready. */}
+                <Text variant="titleSmall">
+                    <Text variant="titleSmall" color="forest">
+                        Trezor Go
+                    </Text>{' '}
+                    addresses
+                </Text>
+
+                <Text variant="hint" color="red" style={applyStyle(hintTextStyle)}>
+                    Good to know
+                </Text>
+                <Text variant="hint" color="gray600" style={applyStyle(hintTextStyle)}>
+                    Trezor Go addresses are best suited for small transfers.
+                </Text>
+                <Text variant="hint" color="gray600" style={applyStyle(hintTextStyle)}>
+                    For large transactions, switch to the Trezor Suite desktop app connected with
+                    your Trezor device to complete the address verification process.
+                </Text>
+                <Text variant="hint" color="gray600" style={applyStyle(hintTextStyle)}>
+                    Learn more about addresses in Trezor Go{' '}
+                    <Text variant="hint" color="forest">
+                        {/* TODO: Create hypertext link here. */}
```

Comment:
Please create task for it or do it now.

Reply:
new issue created https://github.com/trezor/trezor-suite/issues/7464

Reply:
Please put link to that issue to that comment in code.